src/controllers/roleController.js

```diff
@@ -43,4 +46,28 @@ const getRoleId = async (request, response) => {
     );
   }
 };
-export default { getRoleId };
+
+const assignAccomodationManager = async (req, res) => {
+  const reqBody = {
+    accomodationId: req.body.accomodationId,
+    managerId: req.body.managerId
+  }
+  const checkAccomodation = await accomService.getById(reqBody.accomodationId);
+  const checkUser = await userService.findById(reqBody.managerId);
+  if (checkAccomodation == null || checkUser == null) {
```

Comment:
you can use the "!" operator to simplify this line

Reply:
null treated falsy

Reply:
Please address this changes @mucyoalexaxel